### ISSUE TITLE
Just after installation displayed "About" page that doesn't exist

### ISSUE THREAD

**Issue by ghost** [getgrav/grav-theme-mediator#13]
So after fresh install I noticed "About" section in navigation, clicked and got "Page not found". Not sure why this theme comes with this - looks like it's hardcoded?

**Comment by flaviocopes** [comment]
Yes I can see https://github.com/getgrav/grav-theme-mediator/blob/develop/templates/default.html.twig#L31 

Looks like it's "mandatory" for this theme to have an About page, which is present in the Mediator skeleton.

You can override that Twig file in your child theme and change that.

**Comment by ghost** [comment]
@flaviocopes right. So what I did (not perfect) for time-being:

```
{% if page.find('/about') %}
    <a href="{{ base_url_relative }}/about/">About</a>
{% endif %}
```
